pkg/scalers/redis_scaler.go

```diff
@@ -0,0 +1,143 @@
+package scalers
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/go-redis/redis"
+	v2beta1 "k8s.io/api/autoscaling/v2beta1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/metrics/pkg/apis/external_metrics"
+)
+
+const (
+	listLengthMetricName    = "RedisLengthOfQueue"
```

Comment:
`RedisQueueLength` would be a better pick here..

Reply:
@yaron2 thanks! I've updated the metric name to "RedisListLength" now